package.json

```diff
@@ -38,43 +39,45 @@
   },
   "homepage": "https://github.com/VadimDez/ng2-pdf-viewer#readme",
   "dependencies": {
-    "@types/pdfjs-dist": "~2.1.7",
-    "pdfjs-dist": "~2.5.207",
-    "tslib": "^1.10.0"
+    "pdfjs-dist": "~2.6.347",
+    "tslib": "^2.0.0"
+  },
+  "peerDependencies": {
+    "pdfjs-dist": ">=2.6.347"
```

Comment:
Upgraded to `2.6.347` since this is the very 1st version that provides its own typings (`.d.ts`). Haven't encountered breaking changes. Could you verify @VadimDez ?